ajax/libs/octicons/package.json

```diff
@@ -17,18 +17,15 @@
     "svg",
     "octicons"
   ],
-  "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/primer/octicons.git",
-    "fileMap": [
-      {
-        "basePath": "build",
-        "files": [
-          "**/*"
-        ]
-      }
-    ]
-  },
+  "npmName": "octicons",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "/**/!(*.json|*.html)"
```

Comment:
`**/!(*.json|*.html)`, please run git/npm auto-update test before you sent the PR.

Reply:
I did this command and this pattern worked. What is problem with it?

Reply:
```
events.js:183
      throw er; // Unhandled 'error' event
      ^

Error: unexpected end of file
    at Gunzip.zlibOnError (zlib.js:153:15)
```

It also could be my computer's problem, but I suppose relative path `**/!(*.json|*.html)` makes more sense.